messages/agent.create.md

```diff
@@ -1,25 +1,25 @@
 # summary
 
-Create an Agent from an agent spec.
+Create an agent in your org from a local agent spec file.
```

Comment:
This is going to change with the v2 agent creator APIs that will allow "agent preview" behavior, meaning the metadata will not be created in the org but sent back in the response to be previewed by the customer.

Reply:
Gotcha.  I was writing this with the Dec milestone in mind, so I will change it when you implement v2.  Okay?